### PR TITLE
[WIP] Upgrade to Rails 6.0, add webpacker & react

### PR DESCRIPTION
**Pull request by eessex**
DO NOT MERGE

Pair project with @jpotts244

### CODE REVIEW

File: hokusai/development.yml [hunk 1]
```diff
@@ -2,9 +2,9 @@
 version: "2"
 services:
   horizon:
+    env_file: ../.env
```
Comment (joeyAghion) [inline hokusai/development.yml:5]: One problem we've had in other projects is that `hokusai dev` will _fail_ if this file isn't found. If you've found this useful, you might want to add a step to the set-up described in the README.

File: hokusai/development.yml [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
     environment:
       - RAILS_ENV=development
-      - RAILS_LOG_TO_STDOUT=true
```
Comment (joeyAghion) [inline hokusai/development.yml:7]: Why remove this? (Do the defaults change in rails 6? If so the code that references it in the environment files can be removed.)

Comment (eessex) [reply hokusai/development.yml:7]: We reference this in the `build.yml` so it shouldn't need to go into individual environments.

**Comment by eessex** [comment]
Perhaps this is my own bias because I'm inexperienced with haml, but I was frustrated by the difficulty of sharing styles/components/theme in the current setup. By using Palette I hope to significantly speed up the time it takes to build out views and eliminate the need to worry about theming.

**Pull request closed without merging**